.github/workflows/test.yml

```diff
@@ -0,0 +1,49 @@
+---
+name: CI on push release tag
+on:
+  pull_request:
+    branches:
+      - 'main'
+jobs:
+  run-tests:
+    name: Run DTS tests
+    if: ${{ github.head_ref == 'develop' }}
+    runs-on:
+      labels: dts-builder
+    steps:
+      - name: Checkout OSFV repo
+        shell: bash
+        run: |
+          git clone https://github.com/Dasharo/open-source-firmware-validation.git
+      - name: Install requirements
+        shell: bash
+        run: |
+          cd open-source-firmware-validation
+          git submodule update --init --checkout
+          python3 -m virtualenv venv
+          source venv/bin/activate
+          pip install -r requirements.txt
+      - name: Run QEMU
+        shell: bash
+        id: run_qemu
+        run: |
+          ./scripts/ci/qemu-run.sh nographic firmware &
```

Comment:
Why firmware? When using OS there are couple more arguments added to qemu and it uses 4 GB of RAM instead of 1 GB (not sure if it's enough for tests or not):

```
QEMU_PARAMS_OS="-device ich9-intel-hda \
  -device hda-duplex,audiodev=hda \
  -audiodev pa,id=hda,server=${PULSE_SERVER},out.frequency=44100 \
  -object rng-random,id=rng0,filename=/dev/urandom \
  -device virtio-rng-pci,max-bytes=1024,period=1000 \
  -device virtio-net,netdev=vmnic \
  -netdev user,id=vmnic,hostfwd=tcp::5222-:22 \
  -drive file=${HDD_PATH},if=ide"
```
Also related, if this is done to test PR shouldn't you include tested DTS that was built from tested PR? e.g.
`HDD_PATH=/path/to/dts.img ./scripts/ci/qemu-run.sh nographic os`?

Reply:
You're right. I forgot to attach DTS to qemu. I think this will have to be transformed into a reusable workflow called by `ci` and `dev` workflows (before the build directory is removed). I'll think about it.

Reply:
https://github.com/Dasharo/meta-dts/pull/193/commits/192361087c34d912d363de01d87c8de96396d35e